daemons/gptp/common/gptp_cfg.hpp

```diff
@@ -68,6 +69,9 @@ class GptpIniParser
             /*ptp data set*/
             unsigned char priority1;
 
+		/* Clock data set */
```

Comment:
It seems like there might be a tabs vs. spaces conflict in this file; I'm not sure whether it'd be better to just make the change match or if the file itself has a mismatched convention.